File: src/webs/CSSPWebToolsAng/client-apps/src/app/models/MWQMRunModelSiteAndSample.model.ts
import { MWQMRunModel } from "./generated/web/MWQMRunModel.model";
import { MWQMSampleModel } from "./generated/web/MWQMSampleModel.model";
import { MWQMSiteModel } from "./generated/web/MWQMSiteModel.model";

export class MWQMRunModelSiteAndSampleModel {
    MWQMRunModel?: MWQMRunModel;
    MWQMSiteModelAndSampleModelList?: MWQMSiteModelAndSampleModel[];
}

export class MWQMSiteModelAndSampleModel {
    MWQMSiteModel?: MWQMSiteModel;
    MWQMSampleModel?: MWQMSampleModel;
}

File: src/services/CSSPModels/src/MWQMRunModelSiteAndSampleModel.cs
namespace CSSPModels;

[NotMapped]
public partial class MWQMRunModelSiteAndSampleModel
{
    public MWQMRunModel MWQMRunModel { get; set; }
    public List<MWQMSiteModelAndSampleModel> MWQMSiteModelAndSampleModelList { get; set; }

    public MWQMRunModelSiteAndSampleModel()
    {
        MWQMRunModel = new MWQMRunModel();
        MWQMSiteModelAndSampleModelList = new List<MWQMSiteModelAndSampleModel>();
    }
}

